keel-sql/src/main/kotlin/com/netflix/spinnaker/jooq/LocalDateTimeToInstantConverter.kt

```diff
@@ -0,0 +1,17 @@
+package com.netflix.spinnaker.jooq
+
+import org.jooq.impl.AbstractConverter
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneOffset.UTC
+
+class LocalDateTimeToInstantConverter : AbstractConverter<LocalDateTime, Instant>(
+  LocalDateTime::class.java,
+  Instant::class.java
+) {
+  override fun from(databaseObject: LocalDateTime?): Instant? =
+    databaseObject?.toInstant(UTC)
+
+  override fun to(userObject: Instant?): LocalDateTime? =
+    userObject?.atZone(UTC)?.toLocalDateTime()
+}
```

Comment:
Converter that converts between the type JOOQ wants to use by default (`LocalDateTime`) and the one we want to use (`Instant`)